tests/bats/helpers/common.bash

```
@@ -0,0 +1,105 @@
+	assert_equal() {
+		[[ "$1" == "$2" ]] || {
+			echo "# Expected: $1" >&2
+			echo "# Actual:   $2" >&2
+			return 1
+		}
+	}
+fi
+
+setup_temp_dir() {
+	if [[ -z "${BATS_TEST_TMPDIR:-}" ]]; then
+		export BATS_TEST_TMPDIR="$(mktemp -d "${TMPDIR:-/tmp}/bats-test.XXXXXXXXXX")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)